tests/integration/hdf5/test_io.py

```diff
@@ -443,6 +443,11 @@ def test_nwb_version_property(self):
             del io.attrs['nwb_version']
         with NWBHDF5IO(self.path, 'r') as io:
             self.assertTupleEqual(io.nwb_version, (None, None))
+        # check that it works when setting the attribute to a fixed-legnth numpy-bytes string
```

Comment:
```suggestion
        # check that it works when setting the attribute to a fixed-length numpy-bytes string
```

Huh. I wonder why codespell did not catch this.